src/Mono.Android/Java.Interop/TypeManager.cs

```diff
@@ -318,7 +318,7 @@ public static void RegisterType (string java_class, Type t)
 					if (jniFromType != java_class) {
 						managedToJni.Add (t, java_class);
 					}
-				} else {
+				} else if (!ignoreExistingRegistration) {
```

Comment:
We would get that warning a ton in a designer context, the idea is to not clobber our logs with hundreds of them.